content/docs/concepts/components/target.md

```diff
@@ -0,0 +1,26 @@
++++
+title="Targets"
+weight=4
+aliases=[
+    "/docs/using-pack/targets/"
+]
++++
+
+## What is a Target?
+
+A target is the essential information about an operating system and architecture necessary to ensure that binaries will be placed in environments where they can execute successfully.
+A buildpack may specify one or several targets. A build image or run image must specific one target. 
```

Comment:
This statement is confusing to me: `A build image or run image must specific one target.`. Does this mean `[[targets]]` is supposed to be nested inside the `[run]` and `[build]` blocks? I'm more familiar with yaml/json than with toml, so I tend to convert it in order to understand what is going on and it appears to be a builder-wide setting.

given this `builder.toml`:
```toml
[build]
image = "ghcr.io/jericop/build-jammy:target"

[run]
[[run.images]]
image = "ghcr.io/jericop/run-jammy:target"

[[targets]]
os = "linux"
arch = "amd64"
[[targets.distributions]]
name = "ubuntu"
versions = ["18.04", "20.04"]
```

the yaml output is:

```yaml
build:
  image: ghcr.io/jericop/build-jammy:target
run:
  images:
    - image: ghcr.io/jericop/run-jammy:target
targets:
  - os: linux
    arch: amd64
    distributions:
      - name: ubuntu
        versions:
          - "18.04"
          - "20.04"
```

It would be helpful to have a complete example of using `[[targets]]` in a builder toml.